### ISSUE TITLE
哪里修改“predict.py”的测试集路径？

### ISSUE THREAD

**Issue by 15982719597** [milesial/Pytorch-UNet#430]
Hello everyone! 
I am a novice in deep learning. I have a question to ask you to answer. 
Now that I have trained the model, how should I predict my test set? 
Where should the path of the test suite be modified in "predict.py"? 
I have to test the set is a folder of pictures. 
Thank you in advance!

**Comment by qh557** [comment]
您好，我也想咨询一下这个问题，请问您是如何解决的？
先感谢您了
